mountpoint-s3/src/main.rs

```diff
@@ -391,7 +388,6 @@ fn mount(args: CliArgs) -> anyhow::Result<FuseSession> {
     // Placeholder region will be filled in by [create_client_for_bucket]
     let endpoint_config = EndpointConfig::new("PLACEHOLDER")
         .addressing_style(args.addressing_style())
-        .use_fips(args.fips)
```

Comment:
I think we should also remove it from EndpointConfig struct?

Reply:
We should leave that it. In the future, we want to make FIPS endpoints easy to use through a flag when ready.